<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Pathway Commons: A Resource for Biological Pathway Analysis</title>

    <link rel="stylesheet" href="lib/bootstrap-3.3.7-dist/css/bootstrap.min.css">
    <link rel="stylesheet" href="lib/font-awesome-4.5.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/pc.css">

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
          (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
              m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
      ga('create', 'UA-43341809-1', 'pathwaycommons.org');
      ga('send', 'pageview');
      window.ga = ga;
    </script>
  </head>
  <body>

    <!-- BEGIN: site-wrapper -->
    <div class="container site-wrapper">

      <!-- BEGIN: Splash -->
        <!-- Modal -->
        <div class="modal fade" id="infoModal" tabindex="-1" role="dialog" aria-labelledby="infoModal">
          <div class="modal-dialog modal-lg" role="document">
            <div class="modal-content">
              <div class="modal-header">
                <h3 class="modal-title text-center">What's inside version 9</h3>
              </div>
              <div class="modal-body">
                <img src="images/splash.png" class="img-responsive splash" alt="splash">
              </div>
              <div class="modal-footer">
                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
              </div>
            </div>
          </div>
        </div>
      <!-- END: Splash -->

      <!-- BEGIN: jumbotron -->
      <div class="jumbotron">

        <div class="row">
          <div class="hidden-xs col-sm-2">
            <img class="img-responsive brand" alt="pc-logo" src="images/brand/pc_logo.png" width="400" />
          </div>
          <div class="visible-xs">
            <img class="img-responsive brand ghost" alt="pc-logo" src="images/brand/pc_logo.png" />
          </div>

          <div class="col-xs-12 col-sm-10">
            <h1>
              Pathway Commons
            </h1>
            <p>
              Access and discover data integrated from <a class="pc-url" href="pc2/datasources" target="_blank">public pathway and interactions databases</a>.
            </p>
          </div>
        </div>

        <p class="citation text-right">
          <a class="ga ga-publications hidden-xs"
          href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz946/5606621"
          target="_blank">Pathway Commons 2019 Update: integration, analysis and exploration of pathway data.</a>
          <a class="ga ga-publications visible-xs"
          href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz946/5606621"
          target="_blank">Rodchenkov et al. NAR (2019).</a>
        </p>
      </div>
      <!-- END: jumbotron -->

      <!-- BEGIN: Search -->
      <div id="search">
        <form id="pathway-search-form">
          <div class="input-group">
            <input
              type="text"
              id="pathway-search-query-text"
              class="form-control input-lg"
              placeholder="Search the database by keywords or genes" />
            <span class="input-group-addon">
              <button type="submit" class="no-style">
                <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
              </button>
            </span>
          </div>
        </form>
      </div>
      <!-- END: Search -->

      <!-- BEGIN: Training -->
      <div id="training" class="section">
        <div class="page-header">
          <h2>Training <small></small></h2>
        </div>

        <div class="pc-list-group">
          <div class="pc-list-group-item">
            <a href="guide/" target="_blank" class="pc-url ga ga-training">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">Guide</p>
                <p class="pc-list-group-item-description hidden-xs">
                  A pathway analysis online textbook: Workflows provide step-by-step instruction to pathway analysis; primers for deep-dives into concepts.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Pathway analysis online textbook.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_training_guide.png" />
              </div>
            </a>
          </div>
        </div>
      </div>
      <!-- END: Training -->

      <div id="info" class="section">
        <img src="images/splash.png" class="img-responsive splash" alt="splash">
      </div>

      <!-- BEGIN: Data -->
      <div id="data" class="section">
        <div class="page-header">
          <h2>Data <small></small></h2>
        </div>

        <div class="pc-list-group">
          <div class="pc-list-group-item">
            <a href="archives/" target="_blank" class="ga ga-data pc-url">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">File Downloads</p>
                <p class="pc-list-group-item-description hidden-xs">
                    Data prepared in a variety of formats including Biological Pathway Exchange (BioPAX), Simple Interaction Format (sif) and as a Gene Set Database (gmt). Available per datasource.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Data prepared in several formats.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_data_fileDownloads.png" />
              </div>
            </a>
          </div>
        </div>

        <div class="pc-list-group">
          <div class="pc-list-group-item">
            <a href="pc2/" target="_blank" class="ga ga-data pc-url">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">BioPAX Web Services</p>
                <p class="pc-list-group-item-description hidden-xs">
                  'Search' the BioPAX database with full-text; 'Get' an object by URI; use 'Graph' to identify connections and neighborhoods of elements; use 'Traverse' for XPath-like access to the database.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Data prepared in several formats.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_data_webServices.png" />
              </div>
            </a>
          </div>
        </div>

        <div class="pc-list-group">
          <div class="pc-list-group-item">
            <a href="sifgraph/swagger-ui.html" target="_blank" class="ga ga-data pc-url">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">SIF Web Services</p>
                <p class="pc-list-group-item-description hidden-xs">
                  Run fast neighborhood, common stream, etc. graph-theoretical queries on (inferred from the BioPAX model) 
                  our simple genes products and chemicals binary interactions network (using HGNC Symbols and ChEBI IDs).
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Data in the text format (extended SIF).
                </p>
              </div>
              <div class="pc-list-group-item-meta">
              </div>
              <div class="pc-list-group-item-media">
                <img alt="" class="img-responsive" src="images/background_data_webServices.png" />
              </div>
            </a>
          </div>
        </div>

      </div>
      <!-- END: Apps -->


      <!-- BEGIN: Tools -->
      <div id="tools" class="section">
        <div class="page-header">
          <h2>Tools <small></small></h2>
        </div>

        <!-- BEGIN: PCViz -->
        <h3 class="subtitle">PCViz</h3>
        <div id="pcviz">
          <form id="pcviz-form">
            <div class="input-group">
              <input
                type="text"
                id="pcviz-gene-text"
                class="form-control input-lg"
                placeholder="Get details about genes and their interactions (e.g. 'MDM2 TP53')" />
              <span class="input-group-addon">
                <button type="submit" class="no-style">
                  <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
                </button>
              </span>
            </div>
          </form>
        </div>
        <!-- END: PCViz -->


        <h3 class="subtitle">Data visualization and analysis</h3>
        <div class="pc-list-group">
          <div class="pc-list-group-item">
            <a href="http://apps.cytoscape.org/apps/cypath2" target="_blank" class="ga ga-tools">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">CyPath2</p>
                <p class="pc-list-group-item-description hidden-xs">
                  Cytoscape app providing keyword search and retrieval of pathways from Pathway Commons along with advanced filtering and graph queries.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Use Pathway Commons from Cytoscape.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
                <span class="label label-default">Cytoscape</span>
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_tool_cypath2.png" />
              </div>
            </a>
          </div>
          <div class="pc-list-group-item">
            <a href="http://bioconductor.org/packages/release/bioc/html/paxtoolsr.html" target="_blank" class="ga ga-tools">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">PaxToolsR</p>
                <p class="pc-list-group-item-description hidden-xs">
                  R package that faciliates interacting with BioPAX. Supports access to Pathway Commons web services.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Use Pathway Commons data within R.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
                <span class="label label-default">R</span>
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_tool_paxtoolsr.png" />
              </div>
            </a>
          </div>
          <div class="pc-list-group-item">
            <a href="https://github.com/PathwayCommons/chibe" target="_blank" class="ga ga-tools">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">ChiBE</p>
                <p class="pc-list-group-item-description hidden-xs">
                  Chisio BioPAX Editor (ChiBE) is an editing and visualization tool for pathway models represented in BioPAX. Provides access to pathways from Pathway Commons.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  View and edit BioPAX pathway models.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
                <span class="label label-default">Java</span>
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_tool_chibe.png" />
              </div>
            </a>
          </div>
        </div>

        <h3 class="subtitle">Developer resources</h3>
        <div class="pc-list-group">
          <div class="archive pc-list-group-item">
            <a href="http://www.biopax.org/validator" target="_blank" class="ga ga-tools">
              <div class="pc-list-group-item-text">
                <p class="pc-list-group-item-title item-title text-center">BioPAX Validator</p>
                <p class="pc-list-group-item-description hidden-xs">
                  The BioPAX Validator applies custom criteria to identify syntax and semantic errors. Rules originate from the BioPAX Level3 specification.
                </p>
                <p class="pc-list-group-item-description hidden-sm hidden-md hidden-lg">
                  Validate BioPAX using Level3 specification.
                </p>
              </div>
              <div class="pc-list-group-item-meta">
                <span class="label label-default">Web</span>
                <span class="label label-default">Java</span>
              </div>
              <div class="pc-list-group-item-media">
                <img class="img-responsive" src="images/background_tool_biopaxValidator.png" />
              </div>
            </a>
          </div>
        </div>
      </div>
      <!-- END: Apps -->

      <!-- BEGIN: faq -->
      <div id="faq" class="section">
        <div class="page-header">
          <h2>Frequently Asked Questions <small></small></h2>
        </div>

        <!-- BEGIN section: faq-general -->
        <div id="faq-general">
          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-1" aria-expanded="true" aria-controls="faq-1">
              <p class="boost faq-question">What is Pathway Commons?</p>
            </a>
            <div class="collapse" id="faq-1">
              <div class="well">
                <p class="faq-answer">
                  Pathway Commons aims to collect and disseminate biological pathway and interaction data.  Data is collected from
<a class="pc-url" href="pc2/datasources" target="_blank">partner databases</a> and is represented in the
<a href="https://www.biopax.org/" target="_blank">BioPAX</a> standard. By representing data in BioPAX,
Pathway Commons is able to provide a detailed representation of a variety of biological concepts including:
Biochemical reactions; gene regulatory networks; and genetic interactions; transport and catalysis events;
and physical interactions involving proteins, DNA, RNA and small molecules and complexes.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-2" aria-expanded="false" aria-controls="faq-2">
              <p class="boost faq-question">Is it free to use?</p>
            </a>
            <div class="collapse" id="faq-2">
              <div class="well">
                <p>
                  Yes. All of the data provided by Pathway Commons is free! In particular, Pathway Commons distributes pathway information with the intellectual property restrictions of the source database; Only databases that are freely available for academics are included. All of the <a href="https://github.com/PathwayCommons" target="_blank">software that we provide</a> is open-source.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-3" aria-expanded="true" aria-controls="faq-3">
              <p class="boost faq-question">What can I do with this information?</p>
            </a>
            <div class="collapse" id="faq-3">
              <div class="well">
                <ul class="list-unstyled">
                  <li>Researchers</li>
                  <ul>
                    <li>
                      A common practice is to identify pathways that are enriched in gene expression data. To this end,
                      Pathway Commons provides <a class="pc-url" href="archives/" target="_blank">gene set database file downloads</a>
                      for direct use in <a href="http://software.broadinstitute.org/gsea/index.jsp" target="_blank">Gene Set Enrichement Analysis (GSEA)</a>.
                    </li>
                  </ul>
                  <li>Softare developers</li>
                  <ul>
                    <li>
                      Download and incorporate biological pathway data as part of metabolic and gene pathway analysis software in BioPAX Level 3 format. <a href="http://www.biopax.org" target="_blank">Details about the BioPAX format</a>
                    </li>
                  </ul>
                </ul>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-4" aria-expanded="true" aria-controls="faq-4">
              <p class="boost faq-question">How can I access the data?</p>
            </a>
            <div class="collapse" id="faq-4">
              <div class="well">
                <ol class="">
                  <li>Web</li>
                  <ul>
                    <li>
                      <a href="https://apps.pathwaycommons.org/search" target="_blank">Search and view</a> pathways
                    </li>
                    <li>
                      <a class="pc-url" href="pcviz/" target="_blank">Search and view</a> interactions
                    </li>
                  </ul>
                  <li>Desktop software</li>
                  <ul>
                    <li>Retrieve, view and edit Pathway Commons data using the <a href="https://github.com/PathwayCommons/chibe" target="_blank">Chisio BioPAX Editor (ChiBE)</a>
                    </li>
                    <li>Search and analyze Pathway Commons data from <a href="http://apps.cytoscape.org/apps/cypath2" target="_blank">Cytoscape</a>
                    </li>
                    <li>Search and analyze Pathway Commons data using the <a href="http://bioconductor.org/packages/release/bioc/html/paxtoolsr.html" target="_blank">R programming language</a>
                    </li>
                  </ul>
                  <li>Programmatically</li>
                  <ul>
                    <li>Search, retrieve and navigate over Pathway Commons data using our set of <a class="pc-url" href="pc2/" target="_blank">web services</a>
                    </li>
                    <li>Use semantic queries for Pathway Commons data at our <a href="https://rdf.pathwaycommons.org/" target="_blank">SPARQL endpoint</a>
                    </li>
                  </ul>
                </ol>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-5" aria-expanded="false" aria-controls="faq-5">
              <p class="boost faq-question">Does Pathway Commons compete with other pathway databases?</p>
            </a>
            <div class="collapse" id="faq-5">
              <div class="well">
                <p>
                  Pathway Commons does not compete with or duplicate efforts of pathway databases or software tool providers. Pathway Commons will add value to these existing efforts by providing a shared resource for publishing, distributing, querying, and analyzing pathway information. Existing database groups will provide pathway curation, Pathway Commons will provide a mechanism and the technology for sharing. A key aspect of Pathway Commons is clear author attribution. Curation teams at existing databases must be supported by researchers to ensure they can keep performing their valuable work.

                   Pathway Commons enables database providers to share their data in an efficient manner by avoiding duplication of effort and reducing software development overhead.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-general-5" aria-expanded="false" aria-controls="faq-general-5">
              <p class="boost faq-question">How is the data in Pathway Commons represented?</p>
            </a>
            <div class="collapse" id="faq-general-5">
              <div class="well">
                <p>
                  Pathway Commons uses the <a href="https://www.biopax.org" target="_blank">Biological Pathway Exchange (BioPAX)</a> standard to represent data. Pathway databases that make their data available in this format can be imported into Pathway Commons. BioPAX is developed through a collaborative effort by many pathway databases.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-6" aria-expanded="true" aria-controls="faq-6">
              <p class="boost faq-question">What kind of biological concepts are represented?</p>
            </a>
            <div class="collapse" id="faq-6">
              <div class="well">
                <p>
                  Pathway Commons leverages the richness and versatility of <a href="https://www.biopax.org" target="_blank">BioPAX</a> to store data. Details that <em>can</em> be included are biochemical reactions; gene regulatory networks; genetic interactions; proteins, small molecules, DNA, RNA, complexes and their cellular locations; complex assembly and transport; post-translational protein modifications; citations; experimental evidence; and links to other databases e.g. protein sequence annotation. Some information is only available in the downloaded BioPAX files.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-7" aria-expanded="true" aria-controls="faq-7">
              <p class="boost faq-question">How were the pathway data integrated?</p>
            </a>
            <div class="collapse" id="faq-7">
              <div class="well">
                <p>
                  Pathway data are downloaded directly from source databases. Each source has been created differently, some by manual extraction of pathway information from the literature and some by computational prediction. Warehouse data (canonical molecules, ontologies) are converted to BioPAX utility classes and saved as the initial BioPAX model, which forms the foundation for integrating data and for id-mapping. Pathway and binary interaction data (interactions, participants) are normalized next and merged into the database. Original reference molecules are replaced with the corresponding BioPAX warehouse objects.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-8" aria-expanded="true" aria-controls="faq-8">
              <p class="boost faq-question">What happened to the old Pathway Commons site?</p>
            </a>
            <div class="collapse" id="faq-8">
              <div class="well">
                <p>
                  The <a class="pc-url" href="pc/" target="_blank">legacy site</a> has not been updated since 2011 and is not actively maintained.
                </p>
              </div>
            </div>
          </div>

          <div class="qa">
            <a class="ga ga-faq" role="button" data-toggle="collapse" href="#faq-general-6" aria-expanded="false" aria-controls="faq-general-6">
              <p class="boost faq-question">We are a data provider. Why should we export our data to BioPAX?</p>
            </a>
            <div class="collapse" id="faq-general-6">
              <div class="well">
                <p>
                  Benefits of exporting your data to BioPAX and distributing it via Pathway Commons include:
                  <ul>
                    <li>Your data will be used more: Through BioPAX and Pathway Commons, your data can reach more places,
                      including many projects that rely on BioPAX for pathway data import and analysis. We pay attention
                      to ensuring that you are clearly identified as the original data source so that you can receive credit.
                      We log our website usage per data source and provide it back to you for your reporting needs.</li>
                    <li>You will get more feedback and help with quality control: You can use the
                      <a href="http://www.biopax.org/validator">BioPAX validator</a>
                      to check your data against more than a hundred rules. We also automatically and manually check
                      your exported data every release. Users of Pathway Commons often offer great feedback and whenever
                      relevant we pass them back to you.</li>
                    <li>Your data will be compatible with a range of software tools: There are more than 40 active tools
                      that support BioPAX. Do you need web based visualization? You can use PCViz.
                      Do you need graph and pattern searches? There are existing libraries for that.
                      Do you want to use your data in Cytoscape or R? There are multiple apps that support BioPAX.</li>
                    <li>We help you build your website and software tools: You will be able to automatically export
                      your data to many other standard formats through BioPAX to e.g. SBGN, SBML, GSEA, SIF and
                      linked data (RDF). Multiple software components are available to support more rapid application
                      development, such as the powerful <a href="https://www.biopax.org/paxtools">PaxTools</a> Java library.</li>
                    <li>Engage with a community of Pathway Informatics researchers: A key component of the BioPAX community
                      is Pathway Data Providers like you. Through our online forums and face to face meetings, we were able
                      to catalyze excellent convergence and interoperability between pathway databases and software tools.
                      Comparing your data schema against others can give you excellent insights and an opportunity
                      to introduce your ideas to other researchers.</li>
                    <li>We will support your grant applications: Grant agencies often value support for open standard
                      formats as evidenced by several previous grant evaluations. We will provide detailed support letters
                      that explain your involvement and commitment to disseminate your data.
                      We will also provide statistics of your data usage.
                    </li>
                  </ul>
                </p>
              </div>
            </div>
          </div>

        </div> <!-- END SECTION: faq -->
      </div>
      <!-- END: faq -->

      <!-- BEGIN: contact -->
      <div id="contact" class="section">
        <div class="page-header">
          <h2>Contact <small></small></h2>
        </div>

        <p class="boost text-justify">
          Post a question on the <a class="ga ga-contact" href="https://groups.google.com/forum/#!forum/pathway-commons-help/join" target="_blank">Pathway Commons help </a> Google group.
        </p>
        <p class="boost text-justify">
            Keep up-to-date by following the <a class="ga ga-contact" href="https://groups.google.com/forum/#!forum/pathway-commons-announcements/join" target="_blank">Pathway Commons announcements</a> Google group.
        </p>

        <div id="social-media row">
          <ul class="list-inline">
            <li><a class="ga ga-contact ga-social" target="_blank" href="https://twitter.com/pathwaycommons"><i class="fa-twitter fa fa-2x"></i></a></li>
            <li><a class="ga ga-contact ga-social" target="_blank" href="https://github.com/pathwaycommons/"><i class="fa-github fa fa-2x"></i></a></li>
          </ul>
        </div>
      </div>
      <!-- END: contact -->

      <footer>
        <div class="row">
          <div class="col-xs-3 col-sm-2">
            <img class="img-responsive brand hidden-xs" alt="pc-logo" src="images/brand/pc_logo.png" width="250" >
          </div>
          <p class="text-muted col-xs-12 col-sm-10">
            Pathway Commons is a collaboration between the <a target="_blank" href="http://baderlab.org/">Bader Lab</a> at the <a target="_blank" href="https://www.utoronto.ca/">University of Toronto</a>, the <a target="_blank" href="http://www.sanderlab.org/">Sander Lab</a> at the <a target="_blank" href="http://www.dana-farber.org/">Dana-Farber Cancer Institute</a> and <a target="_blank" href="http://hms.harvard.edu/">Harvard Medical School</a> and the Demir Lab at <a target="_blank" href="http://www.ohsu.edu/">Oregon Health & Science University</a>.
          </p>
        </div>
        <br/>
        <div class="row">
          <p class="text-center"><a href="#" title="Back to the top">Top</a></p>
        </div>
      </footer>

    </div>
    <!-- End: site-wrapper   -->

    <!-- Load JS here for greater good =============================-->
    <script src="lib/jquery-3.2.1.min.js"></script>
    <script src="lib/bootstrap-3.3.7-dist/js/bootstrap.min.js"></script>
    <script src="lib/jquery-scrolldepth/jquery.scrolldepth.min.js"></script>
    <script src="js/pc.js"></script>
  </body>
</html>
